.github/workflows/test.yml

```diff
@@ -71,78 +70,27 @@ jobs:
         if: always()
         with:
           name: Integration Tests
-  exporter-tests:
-    name: Exporter tests
-    runs-on: "n1-standard-8-netssd-preempt"
-    timeout-minutes: 45
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/setup-zeebe
-        with:
-          go: false
-      - uses: ./.github/actions/use-ci-nexus-cache
-      - uses: ./.github/actions/build-zeebe
-        with:
-          go: false
-      - name: Create build output log file
-        run: echo "BUILD_OUTPUT_FILE_PATH=$(mktemp)" >> $GITHUB_ENV
-      - name: Maven Test Build
-        run: >
-          mvn -B --no-snapshot-updates
-          -D skipUTs -D skipChecks
-          -D failsafe.rerunFailingTestsCount=3 -D flaky.test.reportDir=failsafe-reports
-          -P extract-flaky-tests
-          -pl :zeebe-elasticsearch-exporter
-          verify
-          | tee "${BUILD_OUTPUT_FILE_PATH}"
-      - name: Duplicate Test Check
-        uses: ./.github/actions/check-duplicate-tests
-        with:
-          buildOutputFilePath: ${{ env.BUILD_OUTPUT_FILE_PATH }}
-      - name: Upload test artifacts
-        uses: ./.github/actions/collect-test-artifacts
-        if: always()
-        with:
-          name: Exporter Tests
-  project-list:
-    # Builds a list of projects where unit tests can be run on hosted runners
-    name: List projects
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - run: sudo apt-get -y install jq
-      - name: Get Project List
-        id: list-projects
-        # Build a json array of module names that can be used for the matrix in `unit-tests`
-        run: >
-          echo "::set-output name=projects::$(find -not -path "./engine/pom.xml" -wholename "./*/pom.xml" -exec dirname {} \; | cut -c 3- | jq -cnR [inputs])"
-    outputs:
-      projects: ${{ steps.list-projects.outputs.projects }}
   unit-tests:
     name: Unit tests
-    needs: project-list
-    strategy:
-      fail-fast: false
-      matrix:
-        project: ${{ fromJson(needs.project-list.outputs.projects) }}
-    runs-on: ubuntu-latest
+    runs-on: "n1-standard-16-netssd-preempt"
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup-zeebe
         with:
           go: false
+      - uses: ./.github/actions/use-ci-nexus-cache
       - uses: ./.github/actions/build-zeebe
         with:
           go: false
-          maven-extra-args: -am -pl ${{ matrix.project }}
       - name: Create build output log file
         run: echo "BUILD_OUTPUT_FILE_PATH=$(mktemp)" >> $GITHUB_ENV
       - name: Maven Test Build
         run: >
           mvn -B --no-snapshot-updates
           -D skipITs -D skipChecks
-          -pl ${{ matrix.project }}
+          -D junitThreadCount=16
```

Comment:
assuming there are modules that can run in parallel would adding `-T 1C` be worth it for speed-up?

Reply:
Maybe, yeah :+1: I've tried to max out CPU via JUnit first but `RandomizedRaftTest` is taking 5 minutes and blocking progress for all others.

Reply:
Done, it removed around 3 minutes